commands/proposal_submission.go

```diff
@@ -464,7 +464,18 @@ func checkLiquidityMonitoring(parameters *protoTypes.LiquidityMonitoringParamete
 		return errs
 	}
 
-	if parameters.TriggeringRatio < 0 || parameters.TriggeringRatio > 1 {
+	tr := float64(0)
+	var err error
+	if len(parameters.TriggeringRatio) > 0 {
+		tr, err = strconv.ParseFloat(strings.TrimSpace(parameters.TriggeringRatio), 64)
```

Comment:
We should be using `num.DecimalFromString()` here so that we're not dealing with floats at all.

I think also for other fields when then value is an empty string we return `ErrIsNotValidNumber` instead of treating it as `"0"`, so we should probably be consistent.

Reply:
Addressed here - https://github.com/vegaprotocol/vega/pull/7222/commits/935f9774a523f334d55e43742ff63338043eb2c4#diff-872de7073d1b2054247d4d9bfcdc63c49d530f5ee501083b4fc7ec970ce13051R467
Thanks!